src/GitHub.App/Models/PullRequestModel.cs

```diff
@@ -131,6 +134,8 @@ public bool HasNewComments
         public DateTimeOffset CreatedAt { get; set; }
         public DateTimeOffset UpdatedAt { get; set; }
         public IAccount Author { get; set; }
+        [AllowNull] public GitReference Base { [return: AllowNull] get; set; }
+        [AllowNull] public GitReference Head { [return: AllowNull] get; set; }
```

Comment:
We already have a `IBranch` somewhere which should probably be representing this information. It'll likely need some tweaking, but it already includes the owner repository, so it can give you access to the clone url.